README.md

```diff
@@ -154,6 +154,10 @@ allowed.
      Restrict to the given address family. If both options are specified no
      addreses will be available and wsdd will exit.
 
+ * `-r`, `--remote`
```

Comment:
As pointed out in #21 (and #24), this feature is not going to be included with its current implementation approach, i.e. providing a static list of announced hosts.